tests/integration/functional/cli/test_secret.py

```diff
@@ -51,7 +54,11 @@ def test_create_secret_with_scope():
     with cleanup_secrets() as secret_name:
         result = runner.invoke(
             secret_create_command,
-            [secret_name, "--test_value=aria", f"--scope={SecretScope.USER}"],
+            [
+                secret_name,
+                '--values={"test_value":"aria"}',
+                f"--scope={SecretScope.USER}",
+            ],
```

Comment:
same here and everywhere else in the tests. You don't need to edit all tests though. Just writing a few new tests to target the new option during create/update is enough.